### ISSUE TITLE
Hello, everything related to "dedicated_server.exe" works fine, but no matter what and where i will change i keep stuck on "Host process was modified/is not supported" Error.. How can i fix that? It's something with this uplay_r1_loader, without it my game runs.

### ISSUE THREAD

**Issue by unixoide** [unixoide/5th-echelon#4]
Hello, everything related to "dedicated_server.exe" works fine, but no matter what and where i will change i keep stuck on "Host process was modified/is not supported" Error.. How can i fix that? It's something with this uplay_r1_loader, without it my game runs.

_Originally posted by @Sylwester1766 in https://github.com/unixoide/5th-echelon/issues/1#issuecomment-1832369547_

**Comment by unixoide** [comment]
@Sylwester1766 Looks like you're using a version of the game which is not supported yet by this project. Can you follow the instructions [here](https://github.com/unixoide/5th-echelon/releases/tag/2023-11-29) so that I can investigate?

**Comment by Sylwester1766** [comment]
> @Sylwester1766 Looks like you're using a version of the game which is not supported yet by this project. Can you follow the instructions [here](https://github.com/unixoide/5th-echelon/releases/tag/2023-11-29) so that I can investigate?

Sure, i'll do that at home, for now i can only say that im using classic original SC:BL from Ubi Connect (Non-Deluxe)

**Comment by Sylwester1766** [comment]
@unixoide 

Tom Clancy's Splinter Cell: Blacklist "Classic" (Ubisoft Connect // Non-Deluxe)
Blacklist_game.exe:
SHA256
HASH: 7FCD3A18D4DCC692719984B07268BD42764108E7DF37049F1490F20929F9925D

**Comment by Sylwester1766** [comment]
On new build i have this one but i think it's the same thing 
![image](https://github.com/unixoide/5th-echelon/assets/49094195/424df4a1-8eae-4aec-bb82-7c102c224f5b)

**Comment by JordanLeich** [comment]
It works for me but maybe its because I'm using the steam version of Blacklist?

**Comment by Sylwester1766** [comment]
> It works for me but maybe its because I'm using the steam version of Blacklist?

Yes i think that i have a problems because of different EXE file than this one on Steam and i think that this mod was developed for Steam exe, @unixoide need to update his mod for executable from Ubisoft Connect. @JordanLeich
